addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -535,6 +536,47 @@ def test_redirect_to_records_nothread(self):
                 self.assertEqual(response.status_code, 200)
                 self.assertNotIn('cids', response.request._cookies)
 
+    def test_call_activity_access_error(self):
```

Comment:
The class is called "TestMultiCompanyRedirect", this is not a redirect test ^^ ... try to avoid putting tests randomly in files ^^ .